src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1 +1,6 @@
-Overview
+%h4.list-group.mb-4 Comments & Request History
+.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
+  = render BsRequestActivityTimeline.new(bs_request: bs_request)
```

Comment:
By convention, component names end in `-Component` as you can read [here](https://viewcomponent.org/guide/getting-started.html).

I'm sure it's just a lapse as you created other components with the correct suffix.

I can add a commit on top of this PR if you want.

Reply:
New commit pushed.